selfdrive/test/test_car_models.py

```diff
@@ -281,6 +281,16 @@ def get_route_logs(route_name):
     'enableCamera': True,
     'enableDsu': True,
   },
+  "e6a24be49a6cd46e|2019-10-24--13-59-04": {
```

Comment:
Seems like a harness was used, so this test should be removed, since when a harness it's used, it will always enable camera and DSU (it does not know the status of the toggle during a replay)

Reply:
OK